.github/workflows/docker-hub-sync.yml

```
@@ -0,0 +1,62 @@
+            ${{ env.IMAGE_NAME }}:latest
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      - name: Update Docker Hub description
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          repository: ${{ env.IMAGE_NAME }}
+          readme-filepath: ./DOCKER.md
+          short-description: "Dockerized Vercel MCP Server - Complete Vercel API management through MCP protocol"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)